src/mono/wasm/test-main.js

```diff
@@ -369,20 +366,9 @@ async function loadDotnet(file) {
         };
     } else if (is_browser) { // vanila JS in browser
         loadScript = async function (file) {
-            const script = document.createElement("script");
-            script.src = file;
-            document.head.appendChild(script);
-            let timeout = 100;
-            // bysy spin waiting for script to load into global namespace
-            while (timeout > 0) {
-                if (globalThis.createDotnetRuntime) {
-                    return globalThis.createDotnetRuntime;
-                }
-                // delay 10ms
-                await new Promise(resolve => setTimeout(resolve, 10));
-                timeout--;
-            }
-            throw new Error("Can't load " + file);
+            globalThis.exports = {}; // if we are loading cjs file
```

Comment:
This is not needed for ES6 case? can we conditionally set it?

Reply:
We don't know that here and there is no harm. When we switch to ES6 as default, all this could go away.